rbac/management/tenant/model.py

```diff
@@ -0,0 +1,526 @@
+#
```

Comment:
Not sold one way or the other, but it was confusing for me when I saw `/management/tenant/model.py` knowing we have `/api/models.py` with a `Tenant` model/class. Trying to think about whether:

- we rename the path here to reflect the model name, so `/management/tenant_mapping/model.py`
- we put this in `/api/models.py` or its own file, alongside `Tenant`

Most of the `management` app models we keep as first-class objects we have APIs for, where the `api` app is more cross-cutting, but I'm curious folks' thoughts here.

Reply:
Yeah I wasn't sure where to put this / what to name it. It's definitely related to Tenants but it pulls in a bunch of other concepts which are management/rbac concepts, not core api concepts (e.g. group, role). Adding this stuff there would create a circular dependency (management depends on api but api depends on management).

Renaming to "tenant_mapping" is probably better than "tenant", so if nothing else that seems like a good idea.

Another option would be to do that and move the bootstrap service stuff to its own module?

Reply:
Yep, exactly what I was thinking too with the overlap with groups/roles, etc., so not making total sense to move to the `Tenant`. I like the idea of renaming and potentially breaking out some of the bootstrap logic.